api/scpca_portal/test/management/commands/test_load_data.py

```diff
@@ -815,7 +819,7 @@ def test_single_cell_metadata(self):
         self.assertIsNone(sample.demux_cell_count_estimate)
         self.assertFalse(sample.has_bulk_rna_seq)
         self.assertFalse(sample.has_cite_seq_data)
-        self.assertEqual(sample.sample_cell_count_estimate, 3417)
+        self.assertEqual(sample.sample_cell_count_estimate, 3432)
```

Comment:
The change here in the assertion stems from a change to the `filtered_cells` attribute in the `SCPCL999990` json file.

Reply:
I feel like at this point it wouldn't hurt to add a comment saying that this line will probably fail when switching test data versions.